src/libmongoc/doc/learn/get/installing.rst

```diff
@@ -30,8 +32,110 @@ management repositories of `common Linux distributions <linux_>`_ and
   For the most up-to-date versions of the C driver libraries, prefer to instead
   :doc:`build from source <from-source>`.
 
+.. seealso::
 
-.. _linux:
+  For a listing and common reference on available packages, refer to
+  :doc:`/ref/packages`.
+
+
+.. index::
+  package managers; Conan
+  package managers; vcpkg
+  pair: installation; package managers
+  :name: installing.pkgman
+
+Cross Platform Installs Using Library Package Managers
+******************************************************
+
+Various library package managers offer |libbson| and |libmongoc| as installable
+packages, including Conan_ and vcpkg_. This section will detail how to install
+using those tools.
+
+.. _conan: https://conan.io/
+.. _vcpkg: https://vcpkg.io/
+
+
+.. index::
+  ! pair: installation; vcpkg
+
+Installing using vcpkg
+======================
+
+.. note::
+  This page will not detail how to get started using vcpkg_. For that, refer to
+  `"Get started with vcpkg" (external)`__
```

Comment:
Is this `(external)` necessary? Suggest removing if not (there are already plenty of links to external sites without this qualifier).

Reply:
I consider it (generally) good practice to annotate links to external domains, but it would be annoying to have to add/update/maintain those on every links, so adding ` (external)` everyone isn't going to work.

So I look into it, and I discover that Sphinx marks every external-linking-`<a>` with an `external` class. Did a quick test, and it seems simple enough to stylize and attach a psuedo-element to all external links at once. :)

I like this solution, but feel free to veto.

Reply:
SGTM, as long as it is consistent across the docs. 👍